tests/test_column_family.rs

```diff
@@ -62,6 +62,17 @@ pub fn test_column_family() {
             Err(e) => panic!("failed to open db with column family: {}", e),
         }
     }
+
+    // should be able to list a cf
+    {
+        let opts = Options::default();
+        let vec = DB::list_cf(&opts, path);
+        match vec {
+            Ok(vec) => println!("list cf: {:?}", vec),
```

Comment:
Could you change the `println!` to an `assert_eq!` that ensures the vec matches what you expect it to be?